client/src/views/v3/StatusView.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { Row, Col, Card } from 'react-bootstrap';
+import ContentPage from 'components/common/ContentPage';
+import CameraStatusCol from 'components/v3/status/CameraStatusCol';
+import WirelessModuleStatusCol from 'components/v3/status/WirelessModuleStatusCol';
+
+// TODO: REMOVE HARD CODING OF PROPS DATA
+const PrimaryCamera = {
+    cameraName: "Primary",
+    isOnline: true,
+    batteryVoltage: 3,
+    ip: "192.168.123.2",
+    videoFeedStatus: "RECORDING",
+};
+
+const SecondaryCamera = {
+    cameraName: "Secondary",
+    isOnline: false,
+};
+
+const FrontWM = {
+    moduleName: "Front WM",
+    isOnline: true,
+    batteryVoltage: 3.1,
+    mqttAddress: "/v3/wireless_module/1/data",
+    sensorData: {
+        CO2: 123,
+        temperature: "21 Degrees",
+        humidity: "99%",
+        accelerometer: "120",
+        gyroscope: 360,
+    }
```

Comment:
You may want to think more about how the data is passed in. My guess is you'll probably want to format the data after it is passed into the component, not before.